src/assets/text/droughtAnnotations.js

```diff
@@ -0,0 +1,76 @@
+export default {
+    timelineEvents: [
+      {
+        id: '1',
```

Comment:
It would be helpful if we're populating these as a group to have the first one or a template annotated with what these values mean and/or their format

Reply:
Great idea - I added comments